resources/zib/zib-LaboratoryTestResult.xml

```diff
@@ -0,0 +1,350 @@
+        <valueSet value="http://nictiz.nl/fhir/ValueSet/TestCode-en-Onderzoek-Codelijsten" />
+      </binding>
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.4" />
+        <comment value="PanelOrBattery" />
+      </mapping>
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.8" />
+        <comment value="TestCode" />
+      </mapping>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="TestDateTime" />
+      <definition value="The date and if possible the time at which the test was carried out." />
+      <alias value="TestDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.13" />
+        <comment value="TestDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.performer">
+      <path value="Observation.performer" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
+          <path value="resolve()" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.performer:performer">
+      <path value="Observation.performer" />
+      <sliceName value="performer" />
+      <short value="Performer" />
+      <definition value="The healthcare provider and/or organization where or by whom the LaboratoryTestResult was performed." />
+      <alias value="Uitvoerder" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner" />
+      </type>
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.35" />
+        <comment value="Performer" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <short value="TestResult" />
+      <definition value="The test result. Depending on the type of test, the result will consist of a value with a unit or a coded value (ordinal or nominal)." />
+      <alias value="TestUitslag" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <type>
+        <code value="CodeableConcept" />

```

Comment:
Why has TestResult two datatypes? I think we should ask zib center what is the best data type for this concept. (see generalmeasurement discussion on teams)

Reply:
Yes, this is one of the open issues I mentioned before. The zib states that this should be "a value with a unit or a coded value (ordinal or nominal)", so Quantity or CodeableConcept -- these should probably be placed on a slice. However, ReferenceRangeUpperLimit and ReferenceRangeLowerLimit are also of datatype ANY, which seems pretty weird (and fundamentally incompatible with FHIR's `.referenceRange`).

Reply:
Relevant zib ticket: https://bits.nictiz.nl/browse/ZIB-981